README.md

```diff
@@ -1 +1,157 @@
-# dagster-lightly-example
\ No newline at end of file
+# Data Pre-processing Pipeline on AWS with Dagster 
+
+Data collection and pre-processing pipelines have become increasingly automated in recent years. The Lightly solution is an intelligent system designed to process raw, unlabeled image data, select the most informative samples for labeling, and mitigate dataset bias.
+
+This guide shows how to write a simple automated data pre-processing pipeline that performs the following steps:
+
+- Download a random video from Pexels.
+- Upload the video to an S3 bucket.
+- Spin up an EC2 instance and start the Lightly Worker waiting for a job to process.
+- Schedule a run for the Lightly Worker and wait till it has finished.
+- Shutdown the instance
+
+Here, the first two steps simulate a data collection process. For this pipeline, you will use Dagster, an open-source data orchestrator for machine learning. It enables building, deploying, and debugging data processing pipelines. Click [here](https://dagster.io) to learn more.
+
+The structure of the tutorial is as follows:
+
+1. [Prerequisites](#1-prerequisites)
+    1. [Setting up the EC2 Instance](#1.1-setting-up-the-ec2-instance)
+    2. [Setting up the S3 Bucket](#1.2-setting-up-the-s3-bucket)
+    3. [Local Setup](#1.3-local-setup)
+2. [Dagster pipeline for processing raw data](#2-dagster-pipeline-for-processing-raw-data)
+    1. [Pipeline breakdown](#2.1-pipeline-breakdown)
+    2. [Running and visualizing the Pipeline](#2.2-running-and-visualizing-the-pipeline)
+    3. [Results](#2.3-results)
+
+
+## 1 Prerequisites
+
+### 1.1 Setting up the EC2 Instance
+
+The first step is to set up the EC2 instance. For the purposes of this tutorial, it’s recommended to pick an instance with a GPU (like the g4dn.xlarge) and the “Deep Learning AMI (Ubuntu 18.04) Version 48.0” AMI. See [this guide](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EC2_GetStarted.html) to get started. Connect to the instance.
+
+Next, you have to [install Lightly Solution](https://docs.lightly.ai/docs/install-lightly). Please follow the instructions. Last, test if the installation on your cloud machine was successful by running the [sanity check](https://docs.lightly.ai/docs/install-lightly#sanity-check):
+
+```
+docker run --shm-size="1024m" --rm -it lightly/worker:latest sanity_check=True
+```
+
+To run the worker remotely, it’s recommended to write a `run.sh` script with default parameters. Saving the command in a bash file is needed to run the worker in an automated way. 
```

Comment:
I would make this more commanding:
> To run the Lightly Worker remotely, add the following to `/home/ubuntu/run.sh`:

(or wherever the script is exactly).

Reply:
Probably we also need to execute `chmod +x`, no?

Reply:
If I'm correct no because SSM already has admin privileges